packages/eslint-plugin/tests/rules/naming-convention.test.ts

```diff
@@ -2224,5 +2224,22 @@ ruleTester.run('naming-convention', rule, {
       ],
       errors: Array(13).fill({ messageId: 'doesNotMatchFormat' }),
     },
+    {
+      code: noFormat`
+        type Foo = {
+          'foo     Bar': string;
+          '': string;
+          '0': string;
+          'foo': string;
+          'foo-bar': string;
+          '#foo-bar': string;
+        };
+
+        interface Bar {
+          'boo-----foo': string;
+        }
+      `,
+      errors: Array(6).fill({ messageId: 'doesNotMatchFormat' }),
```

Comment:
nitpick just to make it clear

```suggestion
      // 6, not 7 because 'foo' is valid
      errors: Array(6).fill({ messageId: 'doesNotMatchFormat' }),
```

Reply:
Oh, yes. I should have commented on it. Thanks for the commit